src/pages/login/Login.tsx

```diff
@@ -0,0 +1,51 @@
+import classNames from 'classnames/bind';
+import Image from 'next/image';
+import loginKakao from 'src/assets/images/loginKakao.png';
+import loginNaver from 'src/assets/images/loginNaver.png';
+
+import styles from './Login.module.scss';
+
+const cx = classNames.bind(styles);
+
+const Login = () => {
+  return (
+    <main className={cx('wrapper')}>
+      <div className={cx('title-wrapper')}>
+        <h1 className={cx('main-title')}>술로그</h1>
+        <h2 className={cx('sub-title')}>전통주를 기록하다</h2>
```

Comment:
단순 부가설명이라면 span 태그를 사용해도 괜찮을 것 같고,
```suggestion
        <h1>술로그</h1>
        <span>전통주를 기록하다</span>
```
h1으로 본다면 스타일링 하기 위해 아래와 같은 구조로 가도 좋을 것 같습니다.
```suggestion
        <h1>
          <span>술로그</span>
          <span>전통주를 기록하다</span>
        </h1>
```
요건 단순 제안일 뿐이니 원세님이 맞다고 생각하시는 방향으로 가면 될 것 같아용!

Reply:
아 h1아래 span을 넣을 수 있군요 감사합니다